app/controllers/application_controller.rb

```diff
@@ -1,3 +1,12 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def configure_permitted_parameters
+    # For additional fields in app/views/devise/registrations/new.html.erb
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:username])
```

Comment:
If you add a new field, you should also change the view registratrion and add a new field (like :           <%= f.input :username,
                      required: true,
                      autofocus: true,
                      input_html: { autocomplete: "PabloPicasso" } %> for exemple) 
                      
                      Same for the edit